.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["node6"]
+  "presets": ["es2015"]
```

Comment:
This is not necessary. Babel is used only for tests, and end users can use whatever babel config they want (provided it's compatible) with the features we use and the targets they want to support.